### DISCUSSION TITLE
Installed the Wyze Bridge Docker Unraid package. Where are all the options for rotation, sound etc?

### DISCUSSION THREAD

**Discussion by superdong69** [mrlt8/docker-wyze-bridge#839] [Q&A]
There's nothing in the GUI to make settings changes, and the only file I can see in the console as having anything to do with settings is config.json. The code is not YAML, and using ROTATE_DOOR=TRUE code to rotate the stream in ANY attempt in config.json renders the container DOA, and I have to reinstall. Can never get a console to open.

Can anyone help?

**Comment by superdong69** [comment]
Anyone?

**Comment by mrlt8** [comment]
You'd probably need to edit and add it to your XML?

https://github.com/mrlt8/docker-wyze-bridge/blob/main/docker-wyze-bridge.xml

**Reply by superdong69** [reply]
I don't have a docker-wyze-bridge.xml in my docker Unraid installed docker from the Unraid appstore

**Reply by superdong69** [reply]
Does anyone have any ideas on this?

**Reply by mrlt8** [reply]
Can someone confirm if the sample [docker-wyze-bridge.xml](https://github.com/mrlt8/docker-wyze-bridge/blob/main/docker-wyze-bridge.xml) works?

**Reply by superdong69** [reply]
How do you apply this to an unraid docker installation?

**Reply by mrlt8** [reply]
Sorry, I don't personally use unraid so can't do any testing or offer much support.
